integration_testing/features/cancel-copy-of-multiple-selections.feature

```diff
@@ -0,0 +1,13 @@
+Feature: Cancel the copy of multiple selections

```

Comment:
Unify this scenario with the previous one into a single `cancel-copy` `.feature` file with 2 scenarios.